### DISCUSSION TITLE
Testing Discussions

### DISCUSSION THREAD

**Discussion by kraih** [mojolicious/mojo#1625] [General]
It's a new GitHub feature, so we are testing it to see how it goes. No plans besides that so far. Have fun! 🎉

**Comment by KrisMatrix** [comment]
Just a random comment.

**Reply by karenetheridge** [reply]
huh, are the replies threaded? (edit: yes they are)

**Comment by scotticles** [comment]
rip google groups.

**Comment by kraih** [comment]
Feel free to write test comments in this thread.

![party](https://user-images.githubusercontent.com/30094/101551208-fb6a3100-39b0-11eb-8b37-7151b8c678ec.gif)

**Reply by scotticles** [reply]
![upvate](https://media.giphy.com/media/Tn5Msrfea6rM4/giphy.gif)

**Reply by scotticles** [reply]
hopefully that helps ;)

**Comment by kraih** [comment]
I bet if you link to issues like #1369 there will be a link back to this thread.

**Reply by kraih** [reply]
Lost that bet... 😭

**Reply by jberger** [reply]
That's too bad, hopefully that's still coming.

**Reply by jberger** [reply]
Unless they're worried that it could get too noisy over there?

**Reply by kraih** [reply]
Perhaps, linking back here from an issue doesn't show anything either though.

**Reply by marcusramberg** [reply]
I’m sure it will eventually.

**Comment by granalberto** [comment]
I want to see if I receive email updates about this discussion.

**Reply by atoomic** [reply]
not sure what looks like reply notifications, testing

**Comment by reneeb** [comment]
Thanks for Mojolicious...

**Comment by granalberto** [comment]
Now I want to reply from email.

**Comment by atoomic** [comment]
testing a simple reply

**Reply by kraih** [reply]
testing reply to your reply

**Reply by karenetheridge** [reply]
it's very noisy with my avatar picture showing up in a reply dialogue box under every single thread...  need compact mode

edit: and emoticon icons under every single post too

**Reply by marcusramberg** [reply]
I think it looks allright. Same as issues, no?

**Comment by yuki-kimoto** [comment]
Hi

This is very good. 

I excited.
